.eslintignore

```diff
@@ -1 +1,2 @@
 mockServices
+src/scenes/home/faq/**/*
```

Comment:
I don't think disabling eslint for the entire file and directories below it is a good idea. Please see the PR comments for a better approach.